gin_integration_test.go

```diff
@@ -115,17 +115,17 @@ func TestWithHttptestWithAutoSelectedPort(t *testing.T) {
 	testRequest(t, ts.URL+"/example")
 }
 
-func TestWithHttptestWithSpecifiedPort(t *testing.T) {
-	router := New()
-	router.GET("/example", func(c *Context) { c.String(http.StatusOK, "it worked") })
-
-	l, _ := net.Listen("tcp", ":8033")
-	ts := httptest.Server{
-		Listener: l,
-		Config:   &http.Server{Handler: router},
-	}
-	ts.Start()
-	defer ts.Close()
-
-	testRequest(t, "http://localhost:8033/example")
-}
+// func TestWithHttptestWithSpecifiedPort(t *testing.T) {
```

Comment:
@tboerger https://travis-ci.org/gin-gonic/gin/builds/210871911 Correct link. Build success on 1.6.x, 1.7.x and 1.8.x but fail in `master`